Doctrine/Orm/Filter/DateFilter.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\ApiBundle\Doctrine\Orm\Filter;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\QueryBuilder;
+use Dunglas\ApiBundle\Api\ResourceInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Filters the collection by date intervals.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ * @author Théo FIDRY <theo.fidry@gmail.com>
+ */
+class DateFilter extends AbstractFilter
```

Comment:
I did not see it at first but you removed the whole `nullOption`? It is no longer possible to specify null first, null last option?